packages/app/src/runner/utils.ts

```diff
@@ -8,7 +8,7 @@ export const REPORTER_ID = 'unified-reporter'
 function getElementById (id: string) {
   const el = document.querySelector<HTMLElement>(`#${id}`)
 
-  if (!el) {
+  if (!el && id !== REPORTER_ID) {
```

Comment:
This isn't a good idea - `getElementById` is generic and used by multiple things, having a check for some specific constant in here definitely is a code smell. Most likely this check should move to whatever function calls `getElementById` that led to this change, probably `getReporterElement`?

Reply:
I think I should remove this helper itself, and use `document.getElementById` directly in `getRunnerElement` and `getReporterElement`, since only `getRunnerElement` actually needs to throw, and this helper function doesn't get used elsewhere.

Reply:
Sure, that works!